counting_prawns.10s.js

```diff
@@ -12,16 +12,7 @@
 
 const https = require('https');
 const bitbar = require('bitbar');
-
-// Configurable params
-const githubConfig = {
-  accessToken: '',
-  username: 'YOUR_USERNAME',
-  owner: 'Marfeel',
-  repos: ['MarfeelXP', 'AliceTenants', 'Gutenberg', 'ProTenants'],
-  color: 'green',
-  alertColor: 'red',
-};
+const githubConfig = require('./config.json');
```

Comment:
Require of NodeJS resolved the import of a JSON file parsing its content:
https://ide.c9.io/dominguezcelada/require-nodejs-jsonfile